src/libs/facets/getPathPrefix.ts

```diff
@@ -0,0 +1,5 @@
+import { i18n } from 'i18next';
+import { getFacet } from './getFacet';
+
+export const getPathPrefix = (i18n: i18n): string =>
+  getFacet() === 'lmel' ? '' : `/${i18n.language}`;
```

Comment:
It seems it was this way before BUT feels akward to  have a `/` in one case and none in the other...

Reply:
It's because we add a whole segment in one case, and not in the other … `/fr/something` versus `/something` …